src/cli.ts

```diff
@@ -20,6 +22,7 @@ const COMMAND_INVOICE = "invoice";
 const COMMAND_RECEIPT = "receipt";
 const COMMAND_PASSPORT = "passport";
 const COMMAND_FINANCIAL = "financial";
+const COMMAND_FR_ID_CARD = "fr_id_card";
```

Comment:
kebab case makes more sense for CLI arguments

```suggestion
const COMMAND_FR_ID_CARD = "fr-id-card";
```